### ISSUE TITLE
How to add md / WYSIWYG area?

### ISSUE THREAD

**Issue by Ezra-Siton-UIX** [Gottwik/Enduro#158]
I find this topic (still open). For now - thier is option to add MD to 
Enduro? (Dont find any docs about this issue beside this post).
https://github.com/Gottwik/Enduro/issues/71
https://github.com/Gottwik/enduro_samples/pull/2

**Why?** for now its really hard to create text-areas without any structure (rich-text). Also **MD vs clients** - how clients/site-managers will write **MD**? (For now i build my own site - but in the future it could be a problem)

Idea like this:
https://quilljs.com/

ANYWAY - why this feature (MD for now) - not build in  - in the core as option (data-type)? (usefull in 99% of the sites)

**Comment by Gottwik** [comment]
Hey Ezra, I totally get the point. The idea is to build a plugin system because enduro is gaining weight. This way, plugging in wysiwyg would be as simple as `enduro install wysiwyg`.

**Comment by Ezra-Siton-UIX** [comment]
**This will be great (I think most of the users will install this plugin)** 

Again - i work with WP, Concrete5, webflow and wix. Although thier endless options  - Most of the work of **clients** is on rich-text-editor (write blog-post / guide / and so on) - so i think this is very important part.

**Comment by rnlambert7** [comment]
I agree that WYSIWYG functionality should be supported. However people like their different Editor plugins, so I think having sticking with the `enduro install wysiwyg` approach could be best. Then its optional. Maybe add a section in the docs about what installing the default one does and what it uses (like the current example pell which I'm now using at your recommendation @Gottwik ) and then some information about how to install your own.

Either that or it is in the core, but the same docs on how to install a new one and disable the original. Options are good either way.

**Comment by Gottwik** [comment]
![image](https://user-images.githubusercontent.com/277311/32410838-6fe3affa-c1ca-11e7-9b31-ad4ce80a83c0.png)

So, I just pushed this, would be awesome if you could try and confirm this works for you.

install quill by `enduro install enduro_quill`. Then you can just use it as admin control type:

```
{
	$test_type: 'quill',
	test: '',
}
```

**Comment by Ezra-Siton-UIX** [comment]
Hi. First if someone read this later - you should first update enduro ([npm update -g enduro](https://docs.npmjs.com/getting-started/updating-global-packages)) than `enduro install enduro_quill`

Gottwik thanks. From CMS side its works fine. But from page view it prints html as text:
```
{
	$test_type: 'quill',
	test: '<h1>hello</h1>',
}
```
`{{test}} `Output in the html as plain text :
```<h1>hello</h1>```

Admin UI::  Height of the text-area should be like `$description_type: 'textarea',`"

**Comment by CosmoMyzrailGorynych** [comment]
@Ezra-Siton-UIX you should use `{{{test}}}` in your templates to output HTML.

[useful docs on Handlebars](http://handlebarsjs.com/)

**Comment by Ezra-Siton-UIX** [comment]
Thanks. Works **great** now. Do you know how can i add options to the toolbar? (Align-right/left) and idea like this. I dont find any setting under enduro (node_modules)
https://quilljs.com/docs/modules/toolbar/

Example:
http://quilljs.com/standalone/full/

**Comment by Gottwik** [comment]
@Ezra-Siton-UIX right 😄, that's phase two, hold tight, I'll get it done.

**Comment by YuraKolesnikov** [comment]
Hi guys! I have a question for you. How do I remove this grey frame around quill toolbar and make it look normal?
![screen shot 2018-01-03 at 16 22 00](https://user-images.githubusercontent.com/28485518/34523934-424f8b34-f0a2-11e7-953c-c25f828d07d9.png)

**Comment by rnlambert7** [comment]
Hi @YuraKolesnikov. I've seen the issue with the frame around the editor a few times in later issues. Here's a quick solve if you need it: https://github.com/Gottwik/Enduro/issues/198

Also, any other `enduro_quill` related issues can created here: https://github.com/Gottwik/enduro_quill/issues

**Comment by rnlambert7** [comment]
@Ezra-Siton-UIX Safe to say the issue can be closed since Gottwik implemented this and brick processors now solve allow custom fields?

**Comment by Ezra-Siton-UIX** [comment]
@rnlambert7 the case is close **beside**: Add some docs about this (great feature --> more users) + RTL support (For now for me - I don't have a way to give clients to write in Hebrew for now).

**Comment by rnlambert7** [comment]
In that case, check out https://www.npmjs.com/package/enduro_pell.

I made that last week as I began with Pell in my project, and the client preferred that so I made a brick to support it. It's almost completely a copy of enduro_quill.

Right now I set it up only with basic features that we are using on our project and then when the project is over, I was going to add better support for it for others use. However you are free to take a look and either write your own, or send a PR. It's very basic right now, but it'd be good to give it the ability to choose certain layouts in the future. (Because of this, I haven't really shared it publicly)

However if you have a better Wysiwyg that supports your needs, I would say duplicate this or quill and create your own.
